### PR TITLE
Patch for Farming Expansion

### PR DESCRIPTION
**Pull request by leugimimi**
## Additions

Patch the small bug from Farming Expansion 

## Reasoning

I altered the stats a bit to make it a little bit weaker, to keep the balance given by CE. It's just a small bettle that exists to be spawned in a event and eat your crops.

## Testing

Check tests you have performed:
- [ x] Game runs without errors
- [x ] (For compatibility patches) ...with and without patched mod loaded
- [x ] Playtested a colony ( and spawned some events)

### CODE REVIEW

**Review by albertbokor — DISMISSED** [review]

**Comment by leugimimi** [comment]
I didn't realize fetching the upstream changes to my fork would make the bot here keep rebuilding the file.

File: Patches/Farming Expansion/Race_CropEaterInsect.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="FE_CropEaterInsect"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>0.00</ArmorRating_Sharp>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Chicken"]/race/baseHealthScale</xpath>
```
Comment (ItsSamGray) [inline Patches/Farming Expansion/Race_CropEaterInsect.xml:39]: Is this intended to patch chickens?

Comment (leugimimi) [reply Patches/Farming Expansion/Race_CropEaterInsect.xml:39]: No, all defnames patched should be "FE_CropEaterInsect". I was making this patch and also fixing the base mod, so I glossed over this error.

Comment (ItsSamGray) [inline Patches/Farming Expansion/Race_CropEaterInsect.xml:39]: We can merge the PR after this gets fixed

**Review by ItsSamGray — CHANGES_REQUESTED** [review]

**Review by N7Huntsman — COMMENTED** [review]
Looks fine other than the swapped in defName.

**Review by ItsSamGray — APPROVED** [review]

**Pull request merged**